### PR TITLE
Update Deploy.json

### PR DESCRIPTION
**Pull request by ejwessel**
## Description

Update deploy.json for goerli and kovan deployments

### CODE REVIEW

File: tools/metadata/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@airswap/constants": "0.3.4",
-    "@airswap/types": "3.5.9",
```
Comment (alicevhenshaw) [inline tools/metadata/package.json]: bump metadata version?

Comment (alicevhenshaw) [reply tools/metadata/package.json]: theres a few places where a `dependencies` is bumped but the package itself isnt

File: source/types/deploys.json [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {
-  "1": "0xa42d613D8B8B6cB9507F46A5dd3FD98276570FD7",
-  "4": "0x42d0f5fE517A0e4aB8be2F3FA7799fB96E30E777"
+  "1":  "0xa42d613D8B8B6cB9507F46A5dd3FD98276570FD7",
```
Comment (alicevhenshaw) [inline source/types/deploys.json]: very picky but why does this file have 2 spaces after the colon?

Comment (poiresel) [reply source/types/deploys.json]: it's been resolved.

File: tools/metadata/package.json (hunk 1 above)
Comment (poiresel) [reply tools/metadata/package.json]: I've updated the versions for the packages.

**Pull request merged**